manager/gui/src/components/alerts/LrAlertCommentReplies.vue

```diff
@@ -0,0 +1,114 @@
+<!--
+* CORTX-CSM: CORTX Management web.
+* Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU Affero General Public License as published
+* by the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU Affero General Public License for more details.
+* You should have received a copy of the GNU Affero General Public License
+* along with this program. If not, see <https://www.gnu.org/licenses/>.
+* For any questions about this software or licensing,
+* please email opensource@seagate.com.
+-->
+<template>
+  <div>
+    <span
+      class="expand-collapse"
+      v-if="replies && replies.length > 0"
+      @click="isRepliesVisible = !isRepliesVisible"
+      >{{ isRepliesVisible ? "Hide Replies" : "Show Replies" }}</span
+    >
+
+    <ul
+      class="comment-replies-container"
+      v-if="isRepliesVisible && replies && replies.length > 0"
+    >
+      <li
+        class="comment-reply"
+        v-for="reply in replies"
+        :key="reply.comment_id"
+      >
+        <div class="created-by">
+          {{ reply.created_by }}
+        </div>
+        <div class="comment-text mt-1">
+          {{ reply.comment_text }}
+        </div>
+        <div class="mt-2 comment-timestamp">
+          {{ formattedTime(new Date(reply.created_time * 1000)) }}
+        </div>
+      </li>
+    </ul>
+    <div class="add-reply">
+      <v-text-field
```

Comment:
this is not as per our design, label should not be there.

Reply:
Replaced the label with the placeholder.